src/company_gui.cpp

```diff
@@ -582,7 +582,7 @@ static const LiveryClass _livery_class[LS_END] = {
 template <SpriteID TSprite = SPR_SQUARE>
 class DropDownListColourItem : public DropDownIcon<DropDownString<DropDownListItem>> {
 public:
-	DropDownListColourItem(int colour, bool masked) : DropDownIcon<DropDownString<DropDownListItem>>(TSprite, PALETTE_RECOLOUR_START + (colour % COLOUR_END), colour < COLOUR_END ? _colour_dropdown[colour] : STR_COLOUR_DEFAULT, colour, masked)
+	DropDownListColourItem(int colour, bool masked) : DropDownIcon<DropDownString<DropDownListItem>>(TSprite, GENERAL_SPRITE_COLOUR(colour % COLOUR_END), colour < COLOUR_END ? _colour_dropdown[colour] : STR_COLOUR_DEFAULT, colour, masked)
```

Comment:
`GENERAL_SPRITE_COLOUR` shows up several times but doesn't appear to originate in this PR. Is it a macro? (It looks like a constant but takes arguments like a function or constructor...)

Reply:
It's a macro that does "PALETTE_RECOLOUR_START + (colour)", it is a bit unrelated but I changed it for consistency.